apps/twap/src/app/conditionalOrder/computeOrderUid.ts

```diff
@@ -0,0 +1,48 @@
+import type { Order } from '@cowprotocol/contracts';
+import { computeOrderUid as _computeOrderUid } from '@cowprotocol/contracts';
+import {
+  OrderParameters,
+  OrderSigningUtils,
+  SupportedChainId,
+} from '@cowprotocol/cow-sdk';
+import { returnNever } from '../types/order';
+import { TypedDataDomain } from 'ethers';
+
+function isOrder(order: Order | OrderParameters): order is Order {
+  return true;
+}
+
+const domainCache: Partial<Record<SupportedChainId, TypedDataDomain>> = {};
```

Comment:
If this doesn't change at all, would it make sense to cache it in a localStorage?

Reply:
If you think it might change, ignore this comment.